hexrd/imageseries/load/imagefiles.py

```diff
@@ -95,14 +95,19 @@ def _load_yml(self):
         self._files = []
         for g in fglob.split():
             self._files += glob.glob(os.path.join(dname, g))
-
+        # !!! must sort due to the non-determinate nature of glob on various
+        #     filesystems.  See https://github.com/HEXRD/hexrd/issues/263
+        self._files = self._files.sort()
```

Comment:
```suggestion 
        self._files.sort()
```

Hi Joel! Sorry I am delayed in reviewing this, I was out on break.

I see my above suggested change was implemented in #369, so that's good.

Also, the sort here changes the output slightly compared to before. This now always sorts the list of files no matter what. So if the user had this, for instance:
```
  files:
    IMAGE-D
    IMAGE-A
```

This would be sorted internally to be `IMAGE-A, IMAGE-D`. Not sure if this matters. If we only wanted to sort the glob rather than the whole list, we could just put `sorted` around the `glob` call like `sorted(glob.glob(...))`.

Otherwise, this looks good!